src/angular/schematics/ng-add/data/input-names.ts

```diff
@@ -4,7 +4,18 @@ export const inputNames: VersionChanges<InputNameUpgradeData> = {
   ['merge' as TargetVersion]: [
     {
       pr: '',
-      changes: [],
+      changes: [
+        {
+          replace: 'title',
+          replaceWith: 'label',
+          limitedTo: { elements: ['sbb-processflow-step', 'sbb-step'] },
+        },
+        {
+          replace: 'i18n-title',
```

Comment:
nice catch :-) should we think at that also for other use cases?

Reply:
Yes, where applicable.